### PR TITLE
Use Raid Config for Message Raid Detection

### PR DESCRIPTION
**Pull request by DPlayer234**
# Changes
Modifies the raid detection for messages so that it works as the help command documents:
- Threshold = minimum amount of messages required to be considered a raid
- Decay = how long messages are retained

Bucket trimming still uses the 30s decay it had previously, so this is effectively the maximum decay time for message raids.
Rather than removing buckets, the MessageDTOs are only flagged to avoid missing parts of a raid.

Untested due to lack of local setup for required services.

# Note
Using separate config values for message raids should still be considered as these are somewhat different from the other raid detection. However, that goes beyond the scope I am comfortable with modifying.

### CODE REVIEW

File: src/Silk/Services/Guild/AutoMod/RaidDetectionService.cs [hunk 1]
```diff
@@ -59,19 +62,24 @@ public async Task<Result> HandleMessageAsync(Snowflake guildID, Snowflake channe
...
-                await _raiders.Writer.WriteAsync(new Raider(raider.GuildID, raider.AuthorID, "Suspected raid: Coordinated message raid detected."));
-            
-            bucket.RemoveAll(r => group.Contains(r));
+            {
+                // Instead of removing buckets, only flag the messages so it doesn't reset the state since the raid might be continuing after this message
+                if (!raider.IsFlagged)
+                {
+                    raider.IsFlagged = true;
+                    await _raiders.Writer.WriteAsync(new Raider(raider.GuildID, raider.AuthorID, "Suspected raid: Coordinated message raid detected."));
+                }
```
Comment (VelvetToroyashi) [inline src/Silk/Services/Guild/AutoMod/RaidDetectionService.cs:81]: Something I should certainly consider adopting in V2...

**Review by VelvetToroyashi — APPROVED** [review]
LGTM, cc @CalebABG

**Review by CalebABG — APPROVED** [review]
Looks good! 🚀

**Comment by VelvetToroyashi** [comment]
Will be added in next deploy.

**Pull request merged**